Employee/WebContent/fieldsEditable.js

```diff
@@ -0,0 +1,6 @@
+function fieldsEditable() {
+	document.getElementsByName("first_name")[0].readOnly = false;
```

Comment:
instead of field name use document.getElementById() function, this is more specific